Update consistency-levels.md

JohnnyDude opened the pull request:
fixed "may be" to "may". 
Original: "the partition sets which be may span regions"
New: "the partition sets which may span regions"

azurecla commented:
Hi __@andyhammar__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

JohnnyDude committed:
Update consistency-levels.md

fixed "may be" to "may". 
Original: "the partition sets which be may span regions"
New: "the partition sets which may span regions"
